src/pages/CountryDetail.tsx

```diff
@@ -107,7 +107,14 @@ export const CountryDetail = () => {
           left="1rem"
           zIndex="20"
         />
-        <Sidebar isOpen={isOpen} onToggle={onToggle} />
+        <Sidebar
+          data={[
```

Comment:
Nie musisz przesyłać props-ów w takiej postaci "data". Powinno to wyglądać tak
<Sidebar isOpen={isOpen} onToggle={onToggle} />